discovery/discovery.go

```diff
@@ -0,0 +1,32 @@
+package discovery
+
+import (
+	"net/url"
+
+	"github.com/livepeer/go-livepeer/core"
+	"github.com/livepeer/go-livepeer/net"
+	"github.com/livepeer/go-livepeer/server"
+
+	"github.com/golang/glog"
+)
+
+type offchainOrchestrator struct {
+	uri   *url.URL
+	bcast server.Broadcaster
+}
+
+func NewOffchainOrchestrator(node *core.LivepeerNode, address string) *offchainOrchestrator {
+	addr := "https://" + address
+	uri, err := url.ParseRequestURI(addr)
+	if err != nil {
+		glog.Error("Could not parse orchestrator URI: ", err)
+		return nil
```

Comment:
why not follow the pattern of returning two return values (o, err) like we do in other places, e.g. `NewLivepeerNode`?

Reply:
We can do that in the next round of updates.